File: C++/FizzBuzz.cpp
/*
This is a classic programming question that almost any beginner programmer should be able to answer.
The objective is to iterate from 1 to n, if the number is divisible by 3, add Fizz to the array, 
if the number is divisible by 5, return "Buzz", and if it is divisible by 3 and 5 return FizzBuzz. 
*/


#include string
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        std::vector<string> strings;
        for( int i = 1; i < n+1; i++){ 
            if (i % 3 ==0 && i % 5 == 0){
                strings.push_back( "FizzBuzz"); 
            }else if (i%3 == 0){
                strings.push_back( "Fizz");  
            }else if (i % 5 ==0){
                strings.push_back( "Buzz"); 
            }else{
                strings.push_back(std::to_string(i)); 
            }
        }
        return strings;
    }
};

File: Python/FizzBuzz.py
'''
This is a classic programming question that almost any beginner programmer should be able to answer.
The objective is to iterate from 1 to n, if the number is divisible by 3, add Fizz to the array, 
if the number is divisible by 5, return "Buzz", and if it is divisible by 3 and 5 return FizzBuzz. 
'''
class Solution:
    def fizzBuzz(n):
        """
        :type n: int
        :rtype: List[str]
        """
        strings = []
        for i in range (1, n+1):
            if (i%3 == 0 and i%5 == 0):
                strings.append("FizzBuzz")
            elif (i%3 == 0):
                strings.append("Fizz")
            elif (i%5 == 0):
                strings.append("Buzz")
            else:
                strings.append(str(i))
        return strings

print(Solution.fizzBuzz(10))